### ISSUE TITLE
ngModel directive is not triggered

### ISSUE THREAD

**Issue by Jeevanandanj** [betsol/angular-input-modified#38]
I've used the version v2.3.1.

ngModel directive is not triggered. So I am always getting modified as false.

**Comment by slavafomin** [comment]
Hello @Jeevanandanj! Could you please create a Plunk to isolate and demonstrate the problem? Make sure to include the latest versions of all the modules. You can fork our demo Plunk if you want: http://plnkr.co/edit/g2MDXv81OOBuGo6ORvdt?p=preview.

**Comment by Jeevanandanj** [comment]
Actually in plunkr and in your sample, I've seen that's working and ng model directive is triggering. But in my application which I've used is not triggering this.

I am using https://github.com/ocombe/ocLazyLoad to load the files.

If i globally declare ng model directive in my application, that's working.But in your plugin that's not working. But ng form directive from your plugin is working.

Here i've confused why the ng model directive alone is not triggering.

**Comment by slavafomin** [comment]
You will have to pinpoint the issue yourself I'm afraid. I will not be able to help without knowing the roots of the problem. Please try to isolate the problem and give me some steps to re-create it, or better create a Plunk with isolated example.

I'm not using lazy-loading for modules myself. Maybe it's a good idea to load our module synchronously with the application. It's functionality is global and the code is minimal, the overhead is almost negligible. You can even [enable the module for only specific forms](https://github.com/betsol/angular-input-modified#excluding-some-fields) if you want to.

**Comment by Jeevanandanj** [comment]
I've renamed extended ng model directive to ng model check and checked. That's actually working. I don't know the actual reason for this. But I am sure the issue is with ng model extended directive.

**Comment by Jeevanandanj** [comment]
Really I don't know the reason. I have tried in plunkr with oclazy loading (http://plnkr.co/edit/s5lzVXQcdbjceUu3puSI?p=preview), but i couldn't reproduce. 

Do I need to include any css?

I've tried with the following steps;
1. Loaded the input modified plugin through ocLazyload in routing.
2. injected the module dynamically. $ocLazyLoad.inject('ngInputModified');

Then I"ve used the same code in my application for sample. But its not working. :-(. Do i missed anything?
